app/models/statement_item.rb

```diff
@@ -0,0 +1,19 @@
+# StatementItem
+#
+# * Is the join table of a many-to-many relation between declarations and
+#   statements
+# * A declaration could have two statement line items max, one for the original
+#   payment and a potential second one if it’s clawed back
+class StatementItem < ApplicationRecord
+  STATES = %w[eligible payable paid voided ineligible awaiting_clawback clawed_back].freeze
```

Comment:
Maybe we should use `Declaration::STATES`, so we have one place.

Reply:
I've checked prod, there is max 2 `statement_line_item` for a declaration.

Reply:
sounds like a good idea 👍 but not any other validation for now